server/http/http.js

```diff
@@ -0,0 +1,12 @@
+import { request } from 'undici';
+
+
+export default class HTTP {
```

Comment:
not sure we should keep the same signature as the previous Meteor's HTTP package, since we're changing the input params and the return type people might be get fooled by thinking they're still using the old package (if they don't check the actual import)..

not sure about how the new signature should look like though :see_no_evil: we might be inspired by `undici` signature, wdyt?

Reply:
I do agree with changing the API, so ppl knows this is different. BTW, I think this can be moved to `server/utils/http` 🤔 (utils doesn't exists in develop tho, so you'll need to create it)

Reply:
what about having the http package exposing verbs (post, get, put etc) and then using named parameters in each one of them to resemble undici's api?

for example: 

```javascript
import { post, get } from './http';
(() => {
  const responsePost = await post({url: 'example.com/endpoint', body: aJSON, headers: someObject});
  const responseGet = await get({url: 'example.com/endpoint', query: queryObject});
})
```

Reply:
That's fine i guess 🤔 that would be easier if you use the latest release of undici (which supports passing `origin` as an object's prop, like you want to do with url)